fstack-protector overwrites fstack-protector=strong and the others are already passed or useless
--
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -108,15 +108,12 @@
 target_compile_options(btop PRIVATE -Wall -Wextra -Wpedantic)
 
 if(NOT APPLE)
-  target_compile_options(btop PRIVATE -fstack-clash-protection)
 endif()
 check_cxx_compiler_flag(-fstack-protector HAS_FSTACK_PROTECTOR)
 if(HAS_FSTACK_PROTECTOR)
-  target_compile_options(btop PRIVATE -fstack-protector)
 endif()
 check_cxx_compiler_flag(-fcf-protection HAS_FCF_PROTECTION)
 if(HAS_FCF_PROTECTION)
-  target_compile_options(btop PRIVATE -fcf-protection)
 endif()
 
 target_compile_definitions(btop PRIVATE
